coralproject/talk
# [CORL-1487] Storage Fallback

## What does this PR do?

Storage used on the parent page can sometimes be restricted (where `localStorage` or `sessionStorage` is nullish). When that occurs, this will fallback to an in-memory storage option.



## What changes to the GraphQL/Database Schema does this PR introduce?

None.



## How do I test this PR?



In incognito window, try to interact with the stream like changing sort options and see the stream still operate but not remember your settings (because it's in-memory, can isn't persisted).
